test/rule-matches/is-visible-on-screen-matches.js

```diff
@@ -0,0 +1,32 @@
+describe('is-visible-on-screen-matches', function () {
+  'use strict';
+
+  var rule;
+  var fixture = document.getElementById('fixture');
+  var fixtureSetup = axe.testUtils.fixtureSetup;
+
+  beforeEach(function () {
+    fixture.innerHTML = '';
```

Comment:
What's this here for? We usually put it in after, although that's not needed anymore either since we have an after in testutils.js that does it for us.